.eslintrc.cjs

```diff
@@ -30,6 +30,12 @@ module.exports = {
     '@typescript-eslint/no-explicit-any': 'warn',
     '@typescript-eslint/prefer-promise-reject-errors': 'warn',
     'unicorn/prefer-add-event-listener': 'warn',
+    '@typescript-eslint/strict-boolean-expressions': [
```

Comment:
Turning this on with the exception of booleans feels very worth it to me. I've written numerous bugs due to how optionals are checked in a truthy/falsey way.